Add support to add custom end-effector attachments for Panda, UR5, and Fetch (#18)
KavrakiLab/vamp: scripts/README.md

Visualization is enabled by `--visualize`, which shows the plan with the PyBullet visualizer.

## `attachments.py`

> Demonstrates planning with end-effector attachments using the same problem as used in `sphere_cage_example.py`.

Reviewer: ```suggestion
Demonstrates planning with end-effector attachments using the same problem as in `sphere_cage_example.py`.
```

> A single sphere is attached to the end-effector of the Franka and carried through the cage.

Reviewer: ```suggestion
A single sphere is attached to the end-effector of the Panda and carried through the cage.
```

> This script has the following arguments that effect the placement of the attached sphere.

Reviewer: ```suggestion
This script has the following arguments that affect the placement of the attached sphere.
```

- `attachment_radius`: Radius of the attached sphere.

> - `attachment_offset`: Offset of the attachment transform along the relative Z-axis (distance from the end-effector).

Reviewer: ```suggestion
- `attachment_offset`: Offset of the attachment transform along the local-frame Z-axis of the end-effector (i.e., its distance from the end-effector).
```
